doc/filtering.rst

```diff
@@ -95,7 +95,7 @@ This section contains functions to filter values during iteration.
    :returns: {gen1, param1, state1}, {gen2, param2, state2}
```

Comment:
It is still incorrect. Please look at the 'Actual mistakes' section in my comment.

Reply:
```
--- a/doc/filtering.rst
+++ b/doc/filtering.rst
@@ -92,10 +92,10 @@ This section contains functions to filter values during iteration.
 .. function:: partition(predicate, gen, param, state)
               iterator:partition(predicate)
 
-   :returns: {gen1, param1, state1}, {gen2, param2, state2}
+   :returns: iterator, iterator
 
```